templates/plugin/python/module.conf.erb

```diff
@@ -1,7 +1,7 @@
   Import "<%= @module %>"
 
   <Module "<%= @module %>">
-  <% @config.sort.each do |key,value| -%>
-    <%= key -%> <%= value -%>
-  <% end %>
+  <%- @config.sort.each do |key,value| -%>
+    <%= key %> <%= value %>
+  <%- end -%>
```

Comment:
I will check how it works for multiple modules.
For only one module I get this config file:

```
# Generated by Puppet
<Plugin "python">
  ModulePath "/usr/lib64/collectd"
  LogTraces false
  Interactive false

  Import "rabbitmq"

  <Module "rabbitmq">
    Host "localhost"
    Password "ppp"
    Port "15672"
    PrefixVhost true
    Realm "RabbitMQ Management"
    Username "uuu"
  </Module>

</Plugin>
```